jmxtrans-output/jmxtrans-output-core/pom.xml

```diff
@@ -53,6 +53,11 @@
 			<artifactId>commons-jexl</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka_2.10</artifactId>
+			<version>0.8.2.1</version>
```

Comment:
Version should be defined in the `dependencyManagement` section of the parent pom. That way we ensure that all modules are using the same version of each dependency.